test/smith/mzcompose.yml

```diff
@@ -20,10 +20,19 @@ services:
       - DIFFERENTIAL_EAGER_MERGE=1000
   smith-fuzz:
     mzbuild: smith-fuzz
-    entrypoint: /bin/bash
     environment:
       - RUST_LOG=smith-fuzz=debug,info
-    command: -c "wait-for-it materialized:6875 --
-      --materialized-host materialized
-      --query-count 100"
+    entrypoint: [smith-fuzz, --materialized-host=materialized]
```

Comment:
Not sure if this still wants to be an entrypoint instead of `command: --materialized-host=materialized` , honestly I find the fact that `command` _doesn't_ include the actual command a little confusing but it's what we do in other places.

So I think this is fine just want to make sure it's an explicit choice.